test/controllers/experiments_controller_test.rb

```diff
@@ -1,10 +1,45 @@
 require 'test_helper'
 
-class ExperimentsControllerTest < ActionController::TestCase
+
+class ExperimeechntsControllerTest < ActionController::TestCase
+  tests ExperimentsController
```

Comment:
Nie rozumiem - co musi być? Wywołanie "tests ExperimentsController"? Dlaczego zmieniłeś nazwę testu na Experimeechnts...?

Jeśli chodzi o "tests" to ustawia ona controller do testowania, jeśli nazwa klasy testu nie mówi o klasie do testowania (ExperimentsControllerTest - ucięcie "Test" daje nam nazwę controllera).

Czy te testy przechodzą? - nie widzę uwierzytelniania... (m.in. ustawienia session[:user])